.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run benchmarks
+
+on:
+  pull_request:
+    paths:
+      - 'EpiAware/**'
+  merge_group:
+    paths:
+      - 'EpiAware/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v1
+      - uses: julia-actions/cache@v1
+      - name: Build EpiAware
+        run: julia --project=EpiAware -e 'using Pkg; Pkg.build()'
+      - name: Build EpiAwarePipeline
+        run: julia --project=EpiAwarePipeline -e 'using Pkg; Pkg.build()'
+      - uses: julia-actions/julia-buildpkg@v1
```

Comment:
@SamuelBrand1 locally this all works now after moving to a repo wide benchmarking suite (as it can't handle subdirectories and no way to hack) but I am really not clear why the develop packages in `Rtwithoutrenewal` can't be found in CI?

Reply:
I've tried a bunch of things here with no succes.